### DISCUSSION TITLE
Center text?

### DISCUSSION THREAD

**Discussion by C8opmBM** [Athou/commafeed#1041] [Q&A] [answered]
Hey, amazing reader!
I've managed to install it via docker compose and reverse-proxied it.
My question is, since I use it exclusively on my pc, is there an option to center the text of the articles?
That's my only annoying gripe.
Thank you!

**Comment by Athou** [comment, marked as answer]
For CommaFeed 2.0, you can use the Custom CSS field on the settings page to adapt the display to your liking.
For CommaFeed 3.0, there's no option yet. It'll be possible when https://github.com/Athou/commafeed/issues/1024 is implemented.
